ccm_web/client/src/pages/MergeSections.tsx

```diff
@@ -127,7 +127,7 @@ function MergeSections (props: CCMComponentProps): JSX.Element {
   }
 
   const isSubAccountAdminOrAccountAdmin = (): boolean => {
-    return isAuthorizedForRoles([RoleEnum['Subaccount admin'], RoleEnum['Account Admin']], props.globals.course.roles, 'MergeSections')
+    return isAuthorizedForRoles([RoleEnum['Subaccount admin'], RoleEnum['Account Admin']], props.globals.course.roles)
```

Comment:
Why was these removed? that seems to be right thing to do. But what did we miss?

Reply:
The string `'MergeSections'` is not used at all, nor were the other stings. From my perspective, it's not necessary, we could always just add a log message for a particular use case if we needed it while we're developing.